.github/workflows/trigger-test-suite.yml

```diff
@@ -19,4 +19,4 @@ jobs:
         steps:
            - name: Trigger Multiverse Test Suite
              run: |
-                gh workflow run -R ${{ secrets.TEST_SUITE_REPO }} ${{ secrets.MULTIVERSE_WORKFLOW }} -f agent_git_ref=${{ github.head_ref || github.ref_name }}
+                gh workflow run -R ${{ secrets.TEST_SUITE_REPO }} ${{ secrets.TEST_SUITE_WORKFLOW }} -f agent_git_ref=${{ github.head_ref || github.ref_name }} -f pr-number=${{ github.event.number }}
```

Comment:
What will `pr-number` be set to when this workflow is triggered by push to `dev` branch?

Reply:
Removed `push` trigger in this commit: d945b847566d97e470eb5d12aff47dbf61902eeb
Triggering this workflow on `push` doesn't make sense as it is primarily concerned with interacting with PRs,